### PR TITLE
Improved wording of French versions

### PR DESCRIPTION
**Pull request by octaveauger**
@adeclosset fyi - please double check one last time.

### CODE REVIEW

File: app/pages/about/jobs/positions/customer-support-french.js [hunk 1]
```diff
@@ -9,48 +9,48 @@ export default class CustomerSupportFrench extends React.Component {
...
+            Les Echos</a> - notre croissance s’accélère en France, où nous avons pour clients
+            des centaines d'entreprises comme Habitat, Mangopay et Zenchef.
           </p>
           <p className='para'>
-            Nous recherchons une personne enthousiaste, aimant résoudre des
-            problèmes élégamment pour rejoindre notre équipe, proposer un
-            service client exceptionnel et contribuer à notre croissance française.
+            Nous recherchons une personne enthousiaste et ingénieuse afin de rejoindre
+            notre équipe, offrir un service client exceptionnel et contribuer à notre
+            croissance française.
```
Comment (comegocardless) [inline app/pages/about/jobs/positions/customer-support-french.js]: @octaveauger 'croissance en France'

File: app/pages/about/jobs/positions/customer-support-french.js [hunk 2]
```diff
@@ -74,48 +74,47 @@ export default class CustomerSupportFrench extends React.Component {
...
           </p>
           <h4 className='u-text-xs u-margin-Bxs u-margin-Tm'>Le/la candidat(e) idéal(e)</h4>
           <ul className='list u-margin-Tm'>
             <li>
-              Vous adorez aider les gens, et êtes intéressés par la technologie.
+              Vous adorez aider les gens, et êtes passionné(e) par la technologie.
             </li>
             <li>
               Vous apprenez rapidement, êtes capable de digérer des informations
               complexes et de les distiller en des réponses claires.
```
Comment (comegocardless) [inline app/pages/about/jobs/positions/customer-support-french.js]: '..en des..' peu pour moi @octaveauger

File: app/pages/about/jobs/positions/customer-support-french.js [hunk 3]
```diff
@@ -74,48 +74,47 @@ export default class CustomerSupportFrench extends React.Component {
...
             <li>
-              Vous êtes organisés, efficaces et attentifs aux détails.
+              Vous êtes organisé(e)s, efficaces et attentifs aux détails.
             </li>
             <li>
-              Vous vous épanouissez dans de petites équipes et aimez collaborer.
+              Vous vous épanouissez dans de petites équipes et aimez aider les autres.
             </li>
             <li>
               Le français est votre langue natale et vous maîtrisez parfaitement le
```
Comment (comegocardless) [inline app/pages/about/jobs/positions/customer-support-french.js]: @octaveauger 'langue maternelle'

**Comment by octaveauger** [comment]
Merging changes to the French copy now.

**Pull request merged**